Quoridor.Core/Pathfinding/PathFinder.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using HavocAndCry.Quoridor.Core.Models;
+
+namespace HavocAndCry.Quoridor.Core.Pathfinding
+{
+    public class PathFinder
+    {
+        // Finds whether a path from player to his finish row exists using Lee (wave) algorithm
+        public static bool IsPathToExit(Player player, GameField gameField)
+        {
+            int[,] labyrinth = new int[gameField.Size, gameField.Size];
+            Queue<WaveStep> queue = new Queue<WaveStep>();
+            queue.Enqueue(new WaveStep(player.Row, player.Column, 1));
+
+            while (queue.Count > 0)
+            {
+                WaveStep waveStep = queue.Dequeue();
+
+                if (waveStep.Row < 0 || waveStep.Row >= gameField.Size || waveStep.Column < 0
+                    || waveStep.Column >= gameField.Size)
```

Comment:
Formatting
Either place all conditions in one row, or each condition in a separate row